src/index.ts

```diff
@@ -46,69 +57,66 @@ interface CodeToolNodes {
  * Code Tool for the Editor.js allows to include code examples in your articles.
  */
 export default class CodeTool implements BlockTool {
-  /** 
-  * Editor.js API
-  */
+  /**
+   * API provided by Editor.js for interacting with the editor's core functionality
+   */
   private api: API;
   /**
-  * Read-only mode flag
-  */
+   * Indicates whether the editor is in read-only mode, preventing modifications
+   */
   private readOnly: boolean;
   /**
-   * CodeTool's placeholder
+   * Placeholder text displayed when there is no code content
    */
   private placeholder: string;
   /**
-   * CodeTool's CSS
+   * Collection of CSS class names used by CodeTool for styling its elements
    */
   private CSS: CodeToolCSS;
   /**
-   * CodeTool nodes
+   * DOM nodes related to the CodeTool, including containers and other elements
    */
   private nodes: CodeToolNodes;
   /**
-  * CodeTool's data
-  */
+   * Stores the current data (code and other related properties) for the CodeTool
+   */
   private _data!: CodeData;
 
   /**
    * Notify core that read-only mode is supported
-   *
-   * @returns {boolean}
+   * @returns true if read-only mode is supported
    */
   static get isReadOnlySupported(): boolean {
     return true;
   }
 
   /**
-   * Allow to press Enter inside the CodeTool textarea
-   *
-   * @returns {boolean}
-   * @public
+   * Allows pressing Enter key to create line breaks inside the CodeTool textarea
+   * This enables multi-line input within the code editor.
+   * @returns true if line breaks are allowed in the textarea
    */
   static get enableLineBreaks(): boolean {
     return true;
   }
 
   /**
-   * @typedef {object} CodeData — plugin saved data
-   * @property {string} code - previously saved plugin code
+   * CodeData — plugin saved data
+   * code - previously saved plugin code
    */
```

Comment:
this comment seems redundant because now we have the `CodeData` interface

Reply:
Okay I will remove it.